packages/experience-builder-sdk/src/blocks/preview/CompositionBlock.tsx

```diff
@@ -22,14 +22,13 @@ import type {
   CompositionNode,
   CompositionUnboundValues,
   CompositionVariableValueType,
-  ExperienceEntry,
   StyleProps,
-} from '../../types';
+} from '@contentful/experience-builder-core/types';
 import { ContentfulContainer } from '@contentful/experience-builder-components';
 import { transformContentValue } from '../../utils/transformers';
 import { resolveDesignComponent } from '../../core/preview/designComponentUtils';
 import { DesignComponent } from '../../components/DesignComponent';
-import { checkIfDesignComponent } from '../../utils/utils';
+import type { ExperienceEntry } from '@contentful/experience-builder-core/types';
```

Comment:
`ExperienceEntry` import can be merged into the `@contentful/experience-builder-core/types` import starting on line 19